io.github.TTimo.GtkRadiant.json

```diff
@@ -0,0 +1,220 @@
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/03_gdkglext-config-h-installation-path.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/glibc2.27-ftbfs.diff"
+        }
+      ]
+    },
+    {
+      "name": "gtkradiant",
+      "no-autogen": true,
+      "sources": [
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/ETPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/ETPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/JAPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/JAPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/Q1Pack/trunk",
+          "revision": "144",
+          "dest": "install/installs/Q1Pack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/Q2Pack/trunk",
+          "revision": "144",
+          "dest": "install/installs/Q2Pack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/Q3Pack/trunk",
+          "revision": "144",
+          "dest": "install/installs/Q3Pack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/QLPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/QLPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/QuetooPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/QuetooPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/STVEFPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/STVEFPack"
+        },
+        {
+          "type": "svn",
+          "url": "https://github.com/Unvanquished/unvanquished-mapeditor-support.git/trunk/build/gtkradiant/",
+          "revision": "207",
+          "dest": "install/installs/UnvanquishedPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/UrTPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/UrTPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/WolfPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/WolfPack"
+        },
+        {
+          "type": "git",
+          "url": "https://github.com/TTimo/GtkRadiant.git",
+          "commit": "6be32bdeecbe1332936a4c43422844691743d75f"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkradiant/revert-uintptr.patch"
+        },
+        {
+            "type": "file",
+            "path": "gtkradiant-Makefile",
```

Comment:
same as above, use buildsystem symple. also `-j8` isn't nice. there is `$FLATPAK_BUILDER_N_JOBS` for that (so that it build faster on builder, but doesn't hammer the poor laptop)